ethereumj-core/src/main/java/org/ethereum/manager/BlockLoader.java

```diff
@@ -99,19 +99,22 @@ public void accept(Block block) {
 
         String fileSrc = config.blocksLoader();
         try {
-            System.out.println("Loading blocks: " + fileSrc);
+            final String blocksFormat = config.getConfig().hasPath("blocks.format") ? config.getConfig().getString("blocks.format") : null;
+            System.out.println("Loading blocks: " + fileSrc + ", format: " + blocksFormat);
 
-            String blocksFormat = config.getConfig().hasPath("blocks.format") ? config.getConfig().getString("blocks.format") : null;
-            if ("rlp".equalsIgnoreCase(blocksFormat)) {
+            if ("rlp".equalsIgnoreCase(blocksFormat)) {     // rlp encoded bytes
                 Path path = Paths.get(fileSrc);
                 // NOT OPTIMAL, but fine for tests
                 byte[] data = Files.readAllBytes(path);
                 RLPList list = RLP.decode2(data);
                 for (RLPElement item : list) {
                     Block block = new Block(item.getRLPData());
                     exec1.push(block);
+                    if (block.getNumber() % 10000 == 0) {
+                        dbFlushManager.flushSync();
+                    }
```

Comment:
This flush shouldn't be there. The flush is now performed automatically and asynchronously on write cache fill. Synchronous flush here will slow import process (and is not good for perf measuring) 
Any reasons behind it?